org.sonarlint.eclipse.core/META-INF/MANIFEST.MF

```diff
@@ -48,9 +48,9 @@ Require-Bundle: org.eclipse.equinox.security,
  org.eclipse.text,
  org.eclipse.lsp4j.jsonrpc,
  com.google.protobuf;bundle-version="3.25.1",
- org.sonarsource.sonarlint.core.sonarlint-rpc-protocol;bundle-version="10.0.0",
- org.sonarsource.sonarlint.core.sonarlint-rpc-java-client;bundle-version="10.0.0",
- org.sonarsource.sonarlint.core.sonarlint-java-client-utils;bundle-version="10.0.0",
- org.sonarsource.sonarlint.core.sonarlint-java-client-legacy;bundle-version="10.0.0"
+ org.sonarsource.sonarlint.core.sonarlint-rpc-protocol;bundle-version="[10.1.0,10.2.0)",
```

Comment:
FYI: This will only allow SLCORE packages with the correct version (currently SLE 10.1 is also targeting SLCORE 10.1). If I try to update to an unsupported version and run Maven locally, this will fail it fast.